c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md_tmpl

```diff
@@ -8,4 +8,4 @@ For this mechanism to work correctly the directive template URL must follow some
 
 In the directive, the template url is written as follow: "templateUrl: {{package}}.baseTemplateUrl + '/<component>/<partial>.html'".
 
-In the main html file, the debug section should contain "{{package}}.templateBaseUrl = '${request.static_url("{{package}}:static-ngeo")}';"
+In the main html file, the debug section should contain "{{package}}.baseTemplateUrl = '${request.static_url("{{package}}:static-ngeo/js/components")}';"
```

Comment:
OK, I understand what you meant; the component name is prepended in the directive.